tests/NHSD.BuyingCatalogue.Ordering.Api.IntegrationTests/Builders/UpdateCatalogueSolutionOrderItemRequestPayloadBuilder.cs

```diff
@@ -0,0 +1,60 @@
+﻿using System;
+using NHSD.BuyingCatalogue.Ordering.Api.IntegrationTests.Requests;
+using NHSD.BuyingCatalouge.Ordering.Api.Testing.Data.Data;
+
+namespace NHSD.BuyingCatalogue.Ordering.Api.IntegrationTests.Builders
+{
+    internal sealed class UpdateCatalogueSolutionOrderItemRequestPayloadBuilder
+    {
+        private DateTime? _deliveryDate;
+        private TimeUnit _estimationPeriod;
```

Comment:
Time unit can be null right? Do you need the '?' a the end?

Reply:
It is a reference type so there is no need to add a '?' as it can be null. DateTime by itself cannot be null & would use a default value.